src/julia.h

```diff
@@ -1116,18 +1150,18 @@ JL_DLLEXPORT jl_value_t *jl_box8 (jl_datatype_t *t, int8_t  x);
 JL_DLLEXPORT jl_value_t *jl_box16(jl_datatype_t *t, int16_t x);
```

Comment:
I believe all the functions to box `bool`, `int8` and `uint8` are also `nosafepoint`.

Reply:
You are correct. In fact I did have that on my original branch, just got missed in this rebase.